packages/shop-mobile-ionic/src/app/pages/+products/+order/takeaway/+page/products-info/products-info.component.html

```diff
@@ -37,6 +42,7 @@
 							}}"
 						>
 							<button
+								*ngIf="isOrderReady()"
```

Comment:
@evereq  We discussed with @tsvetelina-e-y this changes and agree that need to do it also for popup view. Also I think it will be better to disabled the button Got It till the order get needed status "PackagingFinished", not like now to be hidden and appears when get needed status, so you can decide better.

Reply:
@AlishMekliov931 Hm, I see now it's `isOrderReady()` what does it mean ready exactly? Why you think it's always PackagingFinished? what was logic and what is logic after suggested @tsvetelina-e-y changes?

Basically in this PR, it's important to compare what logic was before and what logic will be after @tsvetelina-e-y changes and make sure it's for better, not worse :)

Reply:
@AlishMekliov931 @tsvetelina-e-y I.e. I see a major issue with PR description. It describes not clearly what it was BEFORE PR changes and what it will be AFTER PR changes. We need to see that to decide if "logic" is not broken.

Reply:
Before changes, the user from shop mobile app was able to click button Got It before the order get a status packaging finish, and now he will see this button when the order get status packaging finish from Merchant

Reply:
@AlishMekliov931 @tsvetelina-e-y ok, clear now, completely makes sense I think. Only when it might have issues if some of the order steps will be not used, but we can deal with it later.

Reply:
@evereq What I understand for the issue is that on making order user is presented different view based on whether it is takeaway or delivery. The problem is when case is takeaway. User is able to see button 'Got it' on both the page and popup templates and is actually able to finish the order .. although from merchant app it hasn't been processed yet. I compared the logic between delivery and takeaway templates and saw that in delivery, button for ending the order is hidden with *ngIf="order?.isCompleted && !areIssues" . But when I used isCompleted to check if takeaway order is  completed, it never returns true. Exploring the backend, it is because of this function 

![image](https://user-images.githubusercontent.com/49713089/87315044-a83ca100-c52c-11ea-8015-bc97d8c3f084.png)


 which, if checking order without carrier,  never returns successful status OrderStatus.Delivered. So I decided to add my method isOrderReady() on frontend to check if warehouseStatus of order is at least Packaged .. so I am still not sure how to properly hide the buttons. Also I will make sure to add more descriptive messages to my PRs.